docs/spaces/index.asciidoc

```diff
@@ -117,6 +117,8 @@ You can create a custom experience for users by configuring the {kib} landing pa
 The landing page can route users to a specific dashboard, application, or saved object as they enter each space.
 To configure the landing page, use the `defaultRoute` setting in <<kibana-general-settings, Management > Advanced settings>>.
 
+Examples: `/app/monitoring`, `/app/ml`, `/app/kibana#/dashboards`
```

Comment:
There are all valid examples; however, my concern is whether or not the specific phrasing makes it obvious that only one of the above should be used.

For example, the following is valid:
<img width="671" alt="Screen Shot 2020-01-02 at 2 10 04 PM" src="https://user-images.githubusercontent.com/627123/71696530-1cbaaa00-2d6a-11ea-8e1c-1e762f1c5d06.png">

However, the following isn't:
<img width="664" alt="Screen Shot 2020-01-02 at 2 10 16 PM" src="https://user-images.githubusercontent.com/627123/71696553-29d79900-2d6a-11ea-8286-430d7338b213.png">

Reply:
@ErnestoBezanilla A good approach would be to include the example as an image, instead of as text.  Let me know if you need any help doing this.